docs/schemas/resource/IfcRepresentationResource/Entities/IfcProjectedCRS.md

```diff
@@ -1,59 +1,56 @@
 # IfcProjectedCRS
 
-_IfcProjectedCRS_ is a coordinate reference system of the map to which the map translation of the local engineering coordinate system of the construction or facility engineering project relates. The _MapProjection_ and _MapZone_ attributes uniquely identify the projection to the underlying geographic coordinate reference system, provided that they are well-known in the receiving application. The projected coordinate reference system is assumed to be a 2D or 3D right-handed Cartesian coordinate system, the optional _MapUnit_ attribute can be used determine the length unit used by the map.
+_IfcProjectedCRS_ is a coordinate reference system (CRS) of the map to which the map translation of the local engineering coordinate system of the construction or facility engineering project relates. The projected coordinate reference system is assumed to be a 2D or 3D right-handed Cartesian coordinate system, the optional _MapUnit_ attribute can be used determine the length unit used by the map.
+
+Despite what its name suggests, the _IfcProjectedCRS_ shall be used to represent a **compound coordinate reference system**, meaning a combination of multiple CRS from which a _GeodeticDatum_ and a _VerticalDatum_ can always be unambiguously identified.
```

Comment:
```suggestion
Despite what its name suggests:

* If the referenced _IfcGeometricRepresentationContext_ is 3D, then _IfcProjectedCRS_ shall be a **compound coordinate reference system**, meaning a combination of multiple CRS from which a _GeodeticDatum_ and a _VerticalDatum_ can always be unambiguously identified.

* If the referenced _IfcGeometricRepresentationContext_ is 2D, then _IfcProjectedCRS_ shall be a **projected coordinate reference system**, as the name implies.
```

Reply:
@pjanck better?

Reply:
> If the referenced _IfcGeometricRepresentationContext_ is 2D, then _IfcProjectedCRS_ shall be a **projected coordinate reference system**, as the name implies.

If the referenced _IfcGeometricRepresentationContext_ is 2D, then _IfcProjectedCRS_ can be either a **projected coordinate reference system** or a **compound coordinate reference system**.